scripts/upload-assets.py

```diff
@@ -21,9 +21,10 @@
 args = parser.parse_args()
 version = args.version
 if version == "latest":
-  version = str(subprocess.check_output([
+  version = subprocess.check_output([
     "git", "describe", "--tags", "--abbrev=0"
-  ])).strip()
+  ]).decode('utf-8').strip()
+
```

Comment:
we could drop this in here, too, which would protect both against weird `git describe` results and against weird versions passed as argv[0]:

(also need the `import re`)

```
if not re.match(r'^v\d+\.\d+\.\d+', version):
  sys.exit('error: version %r is not in format v#.#.#' % (version))
```

Reply:
oh wait that's a bad idea - we set the version to SHAs on a most of these

Reply:
i still have a dream that we can use the `go embed` stuff and get rid of a lot of the uploading (tho we'll still need it for snapshots)